### DISCUSSION TITLE
Isolate clients

### DISCUSSION THREAD

**Discussion by fnagua** [ltsp/ltsp#372] [Q&A]
Hello.

First of all, thanks for ltsp job. 

I am a volunteer of a project of repairing old computers and giving them to a school of people in need.

We had a donation of some identical pc's of a big enterprise. Their problem is that they don't have any hard disk. So, our idea was to use the school network to connect all of them to one as a server. The thing is that i want to make isolate clients, it means that each client have his private space.

Is this possible?

**Comment by alkisg** [comment]
Hi @fnagua,

there's no "private space per computer" concept in LTSP. There is a single OS image for all clients.
But there is "private space per user". The home directory, /home/username, is different per user, and can become completely private if you run `sudo chmod 700 /home/*` on the LTSP server.

So the idea is that you create one user account for every student and every teacher, and each one of them have their private space.

If I misunderstood you and you're looking for something else, please clarify...
